app/assets/javascripts/angular/directives/graph_chart.js

```diff
@@ -277,6 +278,19 @@ angular.module("Prometheus.directives").directive('graphChart', ["$location", "W
         rsGraph.series.legend = legend;
         rsGraph.render();
 
+        var drag = new Rickshaw.Graph.DragZoom({
+          graph: rsGraph,
+          opacity: 0.5,
+          fill: 'steelblue',
+          minimumSelection: 25,
```

Comment:
config is up to our preference. minimum selection is the number of pixels that a selection has to exceed in order for the highlighted area to be rendered. this protects against double clicking an area and having it render essentially nothing.